app/navigation/nonsipp/SharesNavigator.scala

```diff
@@ -56,6 +56,9 @@ object SharesNavigator extends JourneyNavigator {
       controllers.nonsipp.shares.routes.CompanyNameRelatedSharesController.onPageLoad(srn, index, NormalMode)
 
     case CompanyNameRelatedSharesPage(srn, index) =>
+      controllers.nonsipp.shares.routes.ClassOfSharesController.onPageLoad(srn, index, NormalMode)
```

Comment:
Will be a different navigation here, FYI @narendrapaduchuri

Reply:
Sure @kberec , I will change it in my branch